### PR TITLE
Bump Capi, Facia, Scanamo & Accompanying fixes (Attempt 2.1)

### PR DESCRIPTION
**Pull request by OllysCoding**
## What does this change?
Bumps scanamo, which then in turn allows us to bump capi and facia clients.

We've done these together as the aim was to bump facia client to get captions on slideshow images. To bump facia, we needed to bump capi, which means we needed to bump scanamo as is has a transient dependency on scala-collection-compat@2.1.2, where as we need 2.6.0.

The [browseDependencyTree](https://github.com/sbt/sbt-dependency-graph) plugin was a great way to find this out. Installing it globally will save lot of time in the future.

[This fix](https://github.com/guardian/frontend/pull/24788/commits/23ce33606b5e2c347bf581577a6fa0ebf26a470d) is interesting in the sense of it wasn't failing during tests... not really sure why.

part of: https://github.com/guardian/frontend/issues/24782 https://github.com/guardian/frontend/issues/24740

---

This is Attempt 2 at this PR (Attempt 1: https://github.com/guardian/frontend/pull/24788), where a new version of the CAPI client was released, allowing us to remove some logic that was overwriting the `format.design` for interactive articles

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]

File: commercial/test/model/LookupTest.scala [hunk 1]
```diff
@@ -87,7 +87,7 @@ import scala.concurrent.duration._
   }
 
   "latestContentByKeyword" should "find content ordered reverse chronologically for an existing keyword" in {
-    val contents = contentsForKeyword("technology/apple")
+    val contents = contentsForKeyword("technology/apps")
```
Comment (OllysCoding) [inline commercial/test/model/LookupTest.scala:90]: Why? 

The `technology/apple` had a (likely rare) occurrence where 2 of the most recent 4 articles where published at the exact same millisecond. Since there's no guaranteed order in this situation, Frontend's "test" and CAPIs order were different, causing the test to fail.

I chose technology/apps as the new tag, but literally any other tag likely won't have this problem.

image


I do question whether this test is needed. Its goal is to test the function we call CAPI with, which should always ask for the 'latest' content. But really it ends up just testing whether CAPI correctly ordered its response on not -> which we should be able to assume it does, with added risks of edge cases like this breaking the test.

Should we just delete this entire test?

Comment (AshCorr) [reply commercial/test/model/LookupTest.scala:90]: I would say its still testing some elements of the Lookup class, although that class mostly seems like its just a wrapper for CAPI.

Maybe sort by ID instead of webPublicationDate?

Comment (OllysCoding) [reply commercial/test/model/LookupTest.scala:90]: I think because CAPI has sorted by webPublicationDate (or there own internal publication date), in usual circumstances that's the only sorting metric we could use that would allow the test to pass the way it's intended to

Comment (mxdvl) [reply commercial/test/model/LookupTest.scala:90]: Maybe we could cc @guardian / digital-cms (if that's the right team?)

**Review by mxdvl — COMMENTED** [review]
Let’s make sure this doesn’t impact https://github.com/guardian/dotcom-rendering/issues/4343 when merged.
